### PR TITLE
chore: adjust omnipool params - leet collection id, fees, tvlcap

### PR DESCRIPTION
**Pull request by enthusiastmartin**

### CODE REVIEW

File: runtime/hydradx/src/lib.rs [hunk 1]
```diff
@@ -805,9 +805,9 @@ impl pallet_uniques::Config for Runtime {
 parameter_types! {
 	pub const LRNA: AssetId = 1;
 	pub const StableAssetId: AssetId = 2;
-	pub ProtofolFee: Permill = Permill::from_rational(3u32,1000u32);
-	pub AssetFee: Permill = Permill::from_rational(3u32,1000u32);
-	pub const TVLCap : Balance= u128::MAX;
+	pub ProtofolFee: Permill = Permill::from_rational(5u32,10000u32);
+	pub AssetFee: Permill = Permill::from_rational(15u32,10000u32);
+	pub const TVLCap : Balance = 100_000_000_000_000_000u128;
```
Comment (jak-pan) [inline runtime/hydradx/src/lib.rs:810]: Isn't this value too small?

Comment (enthusiastmartin) [reply runtime/hydradx/src/lib.rs:810]: increased to 222_222

File: runtime/hydradx/src/lib.rs [hunk 2]
```diff
@@ -805,14 +805,14 @@ impl pallet_uniques::Config for Runtime {
 parameter_types! {
 	pub const LRNA: AssetId = 1;
 	pub const StableAssetId: AssetId = 2;
-	pub ProtofolFee: Permill = Permill::from_rational(3u32,1000u32);
-	pub AssetFee: Permill = Permill::from_rational(3u32,1000u32);
-	pub const TVLCap : Balance= u128::MAX;
+	pub ProtofolFee: Permill = Permill::from_rational(5u32,10000u32);
```
Comment (jak-pan) [inline runtime/hydradx/src/lib.rs:808]: Typo?

Comment (enthusiastmartin) [reply runtime/hydradx/src/lib.rs:808]: what you mean ?

Comment (enthusiastmartin) [reply runtime/hydradx/src/lib.rs:808]: aaah.isee

**Review by lumir-mrkva — APPROVED** [review]

**Pull request merged**